cartography/models/aws/cognito/user_pools.py

```diff
@@ -0,0 +1,73 @@
+from dataclasses import dataclass
+
+from cartography.models.core.common import PropertyRef
+from cartography.models.core.nodes import CartographyNodeProperties
+from cartography.models.core.nodes import CartographyNodeSchema
+from cartography.models.core.relationships import CartographyRelProperties
+from cartography.models.core.relationships import CartographyRelSchema
+from cartography.models.core.relationships import LinkDirection
+from cartography.models.core.relationships import make_target_node_matcher
+from cartography.models.core.relationships import TargetNodeMatcher
+
+
+@dataclass(frozen=True)
+class CognitoUserPoolProperties(CartographyNodeProperties):
+    id: PropertyRef = PropertyRef("Id")
+    name: PropertyRef = PropertyRef("Name")
+
+    # policies: PropertyRef = PropertyRef("Policies")
+    deletion_protection: PropertyRef = PropertyRef("DeletionProtection")
+    # lambda_config: PropertyRef = PropertyRef("LambdaConfig")
+
+    last_modified_date: PropertyRef = PropertyRef("LastModifiedDate")
+    creation_date: PropertyRef = PropertyRef("CreationDate")
+
+    # schema_attributes: PropertyRef = PropertyRef("SchemaAttributes")
+    auto_verified_attributes: PropertyRef = PropertyRef("AutoVerifiedAttributes")
+    alias_attributes: PropertyRef = PropertyRef("AliasAttributes")
+    # verification_message_template: PropertyRef = PropertyRef(
+    #     "VerificationMessageTemplate"
+    # )
```

Comment:
@achantavy Should I avoid storing or drop the `verification_message_template` for now, along with other similar attributes, or does all the data need to be processed?